src/pydap/tests/test_cas_esgf.py

```diff
@@ -0,0 +1,71 @@
+from pydap.client import open_url
+import pydap.net
+from pydap.cas import esgf
+import requests
+import numpy as np
+import os
+import logging
+
+import sys
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
+
+logging.basicConfig(filename='cas_esgf_test.log', level=logging.DEBUG)
+
+
+class TestESGF(unittest.TestCase):
```

Comment:
Could we mark all of these tests in some way that makes them easy to skip? E.g. if some dev doesn't have auth credentials, or one doesn't have a network connection (e.g. while on a plane), it's nice to be able to easily omit the tests that require connectivity.

```python
from nose.plugins.attrib import attr
@attr('auth')
@attr('prod_url')
class TestESGF(unittest.TestCase):
    ...
```